website/docs/cloud-docs/api-docs/run-tasks/run-tasks-integration.mdx

```diff
@@ -180,7 +180,7 @@ Status values other than passed, failed, or running return an error. Both the pa
 
 #### Outcomes Payload Body
 
-A run task result may optionally contain one or more detailed outcomes, which improves result visibility and content in the Terraform Cloud user interface.
+A run task result may optionally contain one or more detailed outcomes, which improves result visibility and content in the Terraform Cloud user interface. In addition to the usual JSON:API fields (see the payload example below), the following attributes define the outcome.
```

Comment:
>  usual JSON:API fields

I don't think we ever really define what are "usual JSON:API fields".  Are you just talking about `type` and `attributes` ?

Reply:
Yeah, I was unsure how to wrap that up in a simple statement without adding (polluting?) the table with 'data' and 'attributes'. Perhaps

> In addition to the JSON:API fields of `type` and `attributes` (see the payload example below), the following properties define the outcome

Reply:
Given it's inside the `attributes` object, I think we don't need that sentence.

```
A run task result may optionally contain one or more detailed outcomes, which improves result visibility and content in the Terraform Cloud user interface. The following attributes define the outcome.
```

Reply:
Good call 👍